tests/PersistentSubscriptionManagement/persistent_subscription_manager.php

```diff
@@ -120,35 +120,32 @@ public function can_describe_persistent_subscription(): void
 
     /**
      * @test
-     * @throws Throwable
      */
-    public function cannot_describe_persistent_subscription_with_empty_stream_name(): void
+    public function cannot_describe_persistent_subscription_with_empty_stream_name(): Generator
     {
-        $this->execute(function () {
+        yield $this->execute(function (): Generator {
```

Comment:
not shure must be `yield from`

Reply:
execute returns a promise so yield is correct

https://github.com/prooph/event-store-client/blob/9e688d64bde3ff6502bb00a8309ff021283f5eef/tests/SpecificationWithConnection.php#L38